CHANGELOG.md

```diff
@@ -6,6 +6,18 @@ All notable changes to [camunda-bpmn-js](https://github.com/camunda/camunda-bpmn
 
 ___Note:__ Yet to be released changes appear here._
 
+## 0.24.0
+* `DEPS`: update to `bpmn-js@11.0.1`
+* `DEPS`: update to `diagram-js@11.1.1`
+* `DEPS`: update modeling dependencies
+
+### Key changes in Modeling
+* `FEAT`: provide templates as replace menu entries ([#207](https://github.com/camunda/camunda-bpmn-js/pull/207))
+* `FEAT`: replace with templates available through feature flag ([#207](https://github.com/camunda/camunda-bpmn-js/pull/207))
```

Comment:
Can you elaborate this? I can't find where said flag is defined and how I can use it.

Reply:
Looked for it in both the PR and the linked issue.